### DISCUSSION TITLE
Crossplay v2.0 for Terraria 1.4.4.7

### DISCUSSION THREAD

**Discussion by Moneylover3246** [Moneylover3246/Crossplay#42] [Announcements]
## Overview
- Introduces support for servers running on Terraria 1.4.4.7
## Changes
- Fixed journey mode support not functioning properly
- Mediumcore & hardcore players may now be blocked from joining a journey mode world when `SoftcoreOnly` is enabled in the TShock config
- The plugin will now throw an exception when it's not compatible with a given server

<hr /><em>This discussion was created from the release <a href='https://github.com/Moneylover3246/Crossplay/releases/tag/2.0'>Crossplay v2.0 for Terraria 1.4.4.7</a>.</em>

**Comment by MaksDragon** [comment]
good job!!

**Comment by GustavoSchip** [comment]
Thanks 🙏

**Comment by DavidZ3319** [comment]
Awesome! Thank you

**Comment by Toopyaaaa** [comment]
it just updated again, can you pls update the plugin again? to 1.4.4.8

**Reply by MaksDragon** [reply]
+, wait to it anD really appreciate
